### DISCUSSION TITLE
Plagiarism detection

### DISCUSSION THREAD

**Discussion by carlosguerrero** [hneemann/Digital#576] [General]
Dear Helmut Neemann,

I'm a teacher in the Balearic Island University, and this academic year we have started using your tool with the students. I think that Digital is a brilliant tool for an academic and teaching context. Congratulations and thank you for your work.

I was wondering (just in case you would know any detail about my question) that if you know any tool that could help teachers to detect plagiarism between the solutions provided by the students in Digital. Since digital stores the circuit project in xml (a text file), I think that it could be possible to compare the text of the .dig files of all the students and detect plagiarism automatically. Do you know any tool to do that with the digital files or any process to do it?

Thank you so much in advance.

**Comment by hneemann** [comment]
To be honest, I only use digital myself for lab exercises and for practice exercises at home. 
I have not yet given students any graded homework that they would have had to submit.
So I can't say much about this question.

**Comment by carlosguerrero** [comment]
In any case, thank you for your great work! It's a very useful tool! Thank you!
